### DISCUSSION TITLE
Feature Request: Be able to Scroll through the list when a data point is clicked on Geomap panel

### DISCUSSION THREAD

**Discussion by raj-shah14** [grafana/grafana#62919] [Dashboards, Panels & Visualizations]
In Geomap panel, i have a region which has a lot more values causing it to go out of page when clicked. If we can add scrolling feature that would help to get the values at the bottom of list.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I believe this is covered by an existing issue https://github.com/grafana/grafana/issues/63324 . Feel free to add more information there.
